addons/hr/static/src/scss/hr.scss

```diff
@@ -1,4 +1,4 @@
-.o_kanban_dashboard.o_hr_department_kanban {
+.o_kanban_dashboard.o_hr_department_kanban .o_kanban_renderer {
```

Comment:
what is the purpose of oe_background_grey?

Reply:
no idea 😅